docs/interpreter/pig.md

```diff
@@ -30,6 +30,8 @@ group: manual
   - MapReduce
   - Tez_Local (Only Tez 0.7 is supported)
   - Tez  (Only Tez 0.7 is supported)
+  - Spark_Local (Only Spark 1.6.x is supported, by default it is Spark 1.6.3)
+  - Spark (Only Spark 1.6.x is supported, by default it is Spark 1.6.3)
 
 ## How to use
 
```

Comment:
it might be helpful to explain that "setup pig" here, to use Local mode set `zeppelin.pig.execType` to `"local"` and for Spark Local Mode set `zeppelin.pig.execType` to `spark_local` - I can see some user might try `spark local` or some variants